Tasks/Src/Task_CheckContactor.c

```diff
@@ -45,6 +47,29 @@ void Task_CheckContactor(void *p_arg) {
             Fault_BitMap |= Fault_ESTOP;
             EnterFaultState();
         }
+
+        // Turn on/off array contactor based on what we receive from controls
+        CAN_ReceiveQueue_Pend(&CanMsg);
+        if (CanMsg.id == ARRAY_CONTACTOR_STATE_CHANGE) {
+          if (CanMsg.payload.data.b) {
+            Contactor_On(ARRAY_CONTACTOR);
+          } else {
+            Contactor_Off(ARRAY_CONTACTOR);
+          }
+        }
+
+        //Send BPS contactor state
+      static uint8_t contactor_state = 0;
+      CanMsg.id = BPS_CONTACTOR_STATE;
```

Comment:
no need to reset a canmsg every time, we're sending the same message periodically